mycompany-seeq-link-connector-myconnector/src/main/java/com/mycompany/seeq/link/connector/DatasourceSimulator.java

```diff
@@ -30,55 +94,177 @@ public String getName() {
         public boolean getStepped() {
             return this.stepped;
         }
+
+        public static class Value {
+            private TimeInstant timestamp;
+            private double measure;
+
+            public Value(TimeInstant timestamp, double value) {
+                this.timestamp = timestamp;
+                this.measure = value;
+            }
+
+            public TimeInstant getTimestamp() {
+                return this.timestamp;
+            }
+
+            public double getMeasure() {
```

Comment:
nit: I think usually this would be an Object not a double. But you can let it double for simplicity.